layout/indicator-data/general/component.tsx

```diff
@@ -343,46 +345,35 @@ const IndicatorChart: FC<ComponentTypes> = ({ className }: ComponentTypes) => {
           && defaultScenario && { scenario: currentScenario }),
       }),
     );
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
-    dispatch,
```

Comment:
I removed some items from the dependency list because they were triggering infinite rerender